code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -111,6 +111,18 @@
 		new /obj/item/reagent_containers/food/snacks/meat/slab/meatproduct(location)
 	return
 
+/datum/chemical_reaction/biomatter_creation
+	name = "Biomatter Creator"
+	id = "biomatter_creation"
+	required_reagents = list(/datum/reagent/blood = 15, /datum/reagent/consumable/nutriment = 30, /datum/reagent/consumable/sugar = 15)
```

Comment:
```suggestion
	required_reagents = list(/datum/reagent/blood = 15, /datum/reagent/consumable/nutriment = 15, /datum/reagent/consumable/sugar = 15)
```

Reply:
yer right ynot, i will change it so it's 1:1:1 ratio